### PR TITLE
(#1525993) Make sure we don't lose stream FDs on journald restart

### PR DESCRIPTION
**Pull request by msekletar**

### CODE REVIEW

**Comment by jsynacek** [comment]
All CI checks are unrelated.

File: src/core/service.c [hunk 1]
```diff
@@ -262,20 +262,28 @@ static void service_fd_store_unlink(ServiceFDStore *fs) {
         free(fs);
 }
 
-static void service_release_resources(Unit *u) {
+static void service_release_fd_store(Service *s) {
+        assert(s);
+
+        log_unit_debug(UNIT(s), "Releasing all stored fds");
```
Comment (jsynacek) [inline src/core/service.c]: This should say UNIT(s)->id. I'm not sure if other patches fix it.

**Review by jsynacek — CHANGES_REQUESTED** [review]

**Review by dtardon — COMMENTED** [review]
LGTM with the exception of the mistake @jsynacek has found.

**Comment by msekletar** [comment]
Fixed. @jsynacek @dtardon please have one more look and give yay or nay. Thanks!

**Review by dtardon — APPROVED** [review]
LGTM now.

**Review by jsynacek — APPROVED** [review]
LGTM.

**Pull request merged**